packages/account/src/Sections/Security/ClosingAccount/closing-account-pending-conditions/closing-account-has-pending-conditions.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { Button, Money, ThemedScrollbars } from '@deriv/components';
+import { CFD_PLATFORMS, formatMoney } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import {
+    TAccounts,
+    TDetailsOfDerivAccount,
+    TDetailsOfDerivXAccount,
+    TDetailsOfMT5Account,
+    TDetailsOfCtraderAccount,
+} from 'Types';
+import ClosingAccountPendingContent from './closing-account-pending-content';
+import ClosingAccountPendingWrapper from './closing-account-pending-wrapper';
+import ClosingAccountPendingBalance from './closing-account-pending-balance';
+import ClosingAccountPendingPositions from './closing-account-pending-positions';
+
+type TDetails = {
+    pending_withdrawals?: Record<string, number>;
+    open_positions?: Record<string, number>;
+    balance?: Record<string, { balance: number; currency: string }>;
+};
+
+type TClosingAccountHasPendingConditionsProps = {
+    details: TDetails | null;
+    onConfirm: () => void;
+};
+
+const getDerivAccount = (client_accounts: TAccounts[], login_id: string) =>
+    client_accounts.find(client_account => client_account.loginid === login_id);
+
+const getCurrentMT5Account = (mt5_login_list: DetailsOfEachMT5Loginid[], login_id: string) =>
+    mt5_login_list.find(account_obj => account_obj.login === login_id);
+
+const getCurrentDxtradeAccount = (dxtrade_accounts_list: TDetailsOfDerivXAccount[], login_id: string) =>
+    dxtrade_accounts_list.find(account_obj => account_obj.account_id === login_id);
+
+const getCurrentCtraderAccount = (ctrader_accounts_list: TDetailsOfCtraderAccount[], login_id: string) =>
```

Comment:
`getCurrentDxtradeAccount` and `getCurrentCtraderAccount` are 100% identical.
Please make one reusable function for both cases:
```
const getCurrentDxOrCtraderAccount = (accounts_list: Array<TDetailsOfDerivXAccount | TDetailsOfCtraderAccount>, login_id: string) =>
    accounts_list.find(account_obj => account_obj.account_id === login_id);
```